service/routes.py

```diff
@@ -23,11 +23,47 @@ def index():
         status.HTTP_200_OK,
     )
 
+######################################################################
+# CREATE A NEW CUSTOMER
+######################################################################
+@app.route("/customers", methods=["POST"])
+def create_customers():
+    """
+    Creates a Customer
+    This endpoint will create a Customer based on the data in the body that is posted
+    """
+    app.logger.info("Request to create an Customer")
+    check_content_type("application/json")
+
+    # Create the account
+    customer = Customer()
+    customer.deserialize(request.get_json())
+    customer.create()
+
+    # Create a message to return
+    message = customer.serialize()
+    # Change to "get customer" when it is made
+    location_url = url_for("create_customers", customer_id=customer.id, _external=True)
```

Comment:
I suspect your test failure is due to this – it need get customer to retrieve the data so that it can be returned as JSON. I'll create it now.